tests/conftest.py

```diff
@@ -1,5 +1,5 @@
-# Copyright 2024 Canonical Ltd.
-# See LICENSE file for licensing details.
+#  Copyright 2025 Canonical Ltd.
```

Comment:
Do we need the extra space? I saw a few others with more space as well

Reply:
Most of the files have two whitespaces, there were some with only one, which were overridden by the copyright update mechanism of my IDE. I use PyCharms and it seems to insert two whitespaces: https://youtrack.jetbrains.com/issue/PY-60543/Two-Spaces-After-Comment-Character-In-Generated-Copyright-Headers-for-Python-Violates-PEP-8